libs/ui/elements/src/lib/user-feedback-item/time-since.pipe.ts

```diff
@@ -0,0 +1,48 @@
+import { Pipe, PipeTransform } from '@angular/core'
+import { TranslateService } from '@ngx-translate/core'
+
+@Pipe({
+  name: 'timeSince',
+  standalone: true,
+})
+export class TimeSincePipe implements PipeTransform {
+  constructor(private translate: TranslateService) {}
+
+  transform(value: Date | string): string {
+    let dateInput: Date
+
+    if (typeof value === 'string') {
+      dateInput = new Date(value)
+      if (isNaN(dateInput.getTime())) {
+        return 'Invalid date'
+      }
+    } else if (value instanceof Date) {
+      dateInput = value
+    } else {
+      return 'Invalid type'
+    }
+
+    const now = new Date()
+    const diff = now.getTime() - dateInput.getTime()
+
+    const minutes = Math.floor(diff / 60000)
+    const hours = Math.floor(diff / 3600000)
+    const days = Math.floor(diff / 86400000)
+    const months = Math.floor(diff / 2629800000)
+    const years = Math.floor(diff / 31557600000)
+
+    if (minutes < 1) {
+      return this.translate.instant('timeSincePipe.lessThanAMinute');
+    } else if (minutes < 60) {
+      return this.translate.instant('timeSincePipe.minutesAgo', { value: minutes, s: minutes > 1 ? 's' : '' });
+    } else if (hours < 24) {
+      return this.translate.instant('timeSincePipe.hoursAgo', { value: hours, s: hours > 1 ? 's' : '' });
+    } else if (days < 30) {
+      return this.translate.instant('timeSincePipe.daysAgo', { value: days, s: days > 1 ? 's' : '' });
+    } else if (months < 12) {
+      return this.translate.instant('timeSincePipe.monthsAgo', { value: months });
+    } else {
+      return this.translate.instant('timeSincePipe.yearsAgo', { value: years, s: years > 1 ? 's' : '' });
+    }
+  }
```

Comment:
This can probably be done by a dependency, although it can also make sense to reimplement it if no dependency is light/fitting enough :)